src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,97 @@
 package pl.javastart.task;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        String fileNameProducts = "products.csv";
+        String fileNameCurrencies = "currencies.csv";
+        File fileProducts = new File(fileNameProducts);
+        File fileCurrencies = new File(fileNameCurrencies);
+        Scanner scannerProducts = null;
+        Scanner scannerCurrencies = null;
```

Comment:
Niepotrzebnie tworzysz to w tej metodzie. Możesz tworzyć te scannery tam, gdzie są używane, czyli w metodach readProductsFromFile i readCurrenciesFromFile